jsapp/js/components/permissions/publicShareSettings.component.tsx

```diff
@@ -64,19 +69,34 @@ class PublicShareSettings extends React.Component<PublicShareSettingsProps> {
 
     return (
       <bem.FormModal__item m='permissions'>
-        <bem.FormModal__item>
-          <Checkbox
-            checked={anonCanView}
-            onChange={this.togglePerms.bind(this, 'view_asset')}
-            label={t('Anyone can view this form')}
+        <bem.FormModal__item m='anonymous-submissions'>
+          <ToggleSwitch
+            checked={anonCanAddData}
+            onChange={this.togglePerms.bind(this, 'add_submissions')}
+            label={t(
+              'Allow submissions to this form without a username and password'
+            )}
           />
+          <a
+            href={envStore.data.support_url + HELP_ARTICLE_ANON_SUBMISSIONS_URL}
+            target='_blank'
+            data-tip={t(
+              'Allow anyone to see this form and add submissions. Click the icon to learn more.'
+            )}
+          >
+            <Icon size='s' name='help' color='storm' />
+          </a>
```

Comment:
It looks like this could get pulled out into its own component - the only difference (that I can tell) between this and [this bit](https://github.com/kobotoolbox/kpi/pull/4728/files/80383614fe58675f5372b19b163087127c1971a8..f6db90642ae3efd0d10d612ac52cc0ac33a84f48#diff-419411dcab5d5a1abd3e658254891eab203590e61a388f4a173c4f090bd20501R428-R445) appear to be the `checked` and `onChange` props.